### ISSUE TITLE
Splitting a msg

### ISSUE THREAD

**Issue by Cing2** [QBFreak/QDTTS#2]
In server.lua line 119 you have this: `id,msg = rednet.receive("QDTTS")

  message = {}
  count = 0
  for i in string.gmatch(msg, "%S+") do
    count = count + 1
    message[count] = i
  end`
I have used your code to make the inventory server parted or at least i am trying to.
But i find it weird that you have the count+1 before the message[count] so that the first slot will always be empty i don't know how your code handels the rest of it but i just wanted to point this out.

**Comment by QBFreak** [comment]
Aren't all subscripts in Lua actually more like keys in key value pairs? If so, there's technically no 'first slot' to be ignored. Try putting something in arr[1] and using the function to print arrays (drawing a blank ATM), I bet you'll find that's the only one it shows you. Skip ahead to arr[5] and do it again. Iirc, you'll just get values for 1 and 5, nothing else. It's less that the others are empty and more that they don't exist. Unless I'm remembering wrong, then forget everything I just said :)

Regardless, I started at 1 for a reason although I can't recall what at the moment. Take a look where the values of message get used, perhaps there I found it easier to start at 1 than 0.

**Comment by Cing2** [comment]
I haven't looked in your code to much but if you also use 1 for the first
message then there is no problem.
But when i used your code i started at 0 so i didn't got well.

2016-04-17 3:11 GMT+02:00 Jason Hill notifications@github.com:

**Comment by QBFreak** [comment]
Now that I'm back at a PC and not squinting at my phone, I took a quick peek at [`server.lua line 126`](https://github.com/QBFreak/QDTTS/blob/master/server.lua#L126) where message gets used and I definitely start at 1. `message[1]` is the command, `message[2]` is the sending turtle name. Any additional parameters vary depending on the command. I don't recall why I started at 1, I may have originally been planning on putting a count in 0.
